BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.java

```diff
@@ -183,7 +201,7 @@ boolean canPerformBrowserSwitch(FragmentActivity activity, @BraintreeRequestCode
         return manifestValidator.isUrlSchemeDeclaredInAndroidManifest(applicationContext, urlScheme, klass);
     }
 
-    ActivityInfo getManifestActivityInfo(Class klass) {
+    <T> ActivityInfo getManifestActivityInfo(Class<T> klass) {
```

Comment:
Curious - why does this need to be generic now?

Reply:
This one was a lint error. I think it doesn't like when `Class` has no associated type since it's generic. The one above has a similar kind of syntax figured it was at least reducing the number of lint errors.